src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private String color;
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/113d5a10-ec56-4dca-a962-1cfc9df9d604)

Reply:
not fixed